src/HonzaBotner.Discord.Services/Commands/ReminderCommands.cs

```diff
@@ -39,10 +38,10 @@ IReminderManager reminderManager
             _reminderManager = reminderManager;
         }
 
-        [GroupCommand]
```

Comment:
Reason for removing this?

Reply:
> New reminder is no longer group command - Made it confusing when people were just trying to learn how to use them and it immediately counted towards cooldown, so sometimes they even didn't manage to create one.

When you enter `::reminder` now it displays help on how to use it instead of immediately failing for not enough arguments and counting towards cooldown. I might even expand that description to include examples as many people just fail when creating reminders.